src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -204,13 +220,15 @@ public ActionResult<List<Course.Membership>> GetCourseMemberships(Guid courseId,
 
 
         [HttpDelete]
-        [Route("{courseId}/Memberships/{userId}")]
-        public ActionResult<Course> DeleteCourseMembership(Guid courseId, Guid userId)
```

Comment:
I think the last merge from master went wrong.
The route should be "{courseId}/Memberships/{userId}".
See #39